logging/src/main/scala/com/ovoenergy/effect/Tracing.scala

```diff
@@ -66,29 +66,29 @@ object Tracing {
     }
 
   /**
-    * Turn a Traced[F, A] into an F[A]
-    * by running it with an initial empty trace context
-    */
+   * Turn a Traced[F, A] into an F[A]
+   * by running it with an initial empty trace context
+   */
   def dropTracing[F[_]: Monad]: Traced[F, ?] ~> F =
     new (Traced[F, ?] ~> F) { def apply[A](t: Traced[F, A]): F[A] = t.runEmptyA }
 
   /**
-    * The following functions are syntax sugar for various operations on the trace context
-    * most notably trace, which takes a function requiring tags and provides them..
-    * useful for logging
-    */
-  def token[F[_] : Tracing : Functor]: F[TraceToken] =
+   * The following functions are syntax sugar for various operations on the trace context
+   * most notably trace, which takes a function requiring tags and provides them..
+   * useful for logging
+   */
+  def token[F[_]: Tracing: Functor]: F[TraceToken] =
     Tracing[F].get.map(_.token)
 
-  def putToken[F[_] : Tracing : Monad](token: TraceToken): F[Unit] =
+  def putToken[F[_]: Tracing: Monad](token: TraceToken): F[Unit] =
     Tracing[F].get.flatMap(c => Tracing[F].put(c.copy[Id](token = token)))
 
-  def mdc[F[_] : Tracing: Functor]: F[Tags] =
+  def mdc[F[_]: Tracing: Functor]: F[Tags] =
     Tracing[F].get.map(c => c.mdc.updated("traceToken", c.token.value))
 
   def trace[F[_]: FlatMap](fn: Tags => F[Unit])(implicit t: Tracing[Traced[F, ?]]): Traced[F, Unit] =
     mdc[Traced[F, ?]].flatMapF(fn)
 
-  def putMdc[F[_] : Tracing : Monad](tags: (String, String)*): F[Unit] =
+  def putMdc[F[_]: Tracing: Monad](tags: (String, String)*): F[Unit] =
```

Comment:
what have you done, did you update your local settings or it was always like this but the rules werent applied?

Reply:
I think we just haven't run scalafmt on the project for ages 🤷‍♂